target/msm8916/init.c

```diff
@@ -196,6 +254,10 @@ void target_init(void)
 
 	spmi_init(PMIC_ARB_CHANNEL_NUM, PMIC_ARB_OWNER_ID);
 
+#if WITH_LK2ND
+	lk2nd_init();
+#endif
```

Comment:
I wonder if we can put this in some LK core file instead to avoid patching every SoC? Maybe directly before `target_init()`.